roles/edpm_nova/templates/02-nova-host-specific.conf.j2

```diff
@@ -1,2 +1,3 @@
 [DEFAULT]
 my_ip = {{ ctlplane_ip }}
+host = {{ canonical_hostname }}
```

Comment:
as much as i dislike ever setting this i think this is reasonable.
my expectation is if we decided to go down this path we will also set this in all the other series in the same pr but perhaps in different commits in this pr to ensure we are aligned.

this may impact the telemetry operator too.

Reply:
for addtional context 

there have been numerous attempts since osp 10 to ensure that the hypervisor hostname and host values are the FQDN because of TLS-e requirements. this is meant to be applied regardless of if TLS-e is enabled. triple i belive technically can be configured such that hostname are uses but our product should have been using FQDNS for many years now.

unfortunately this is something that has been broken repataly breakuign upgrades and scale-in/out 
https://bugzilla.redhat.com/show_bug.cgi?id=2181566
https://bugzilla.redhat.com/show_bug.cgi?id=1657692
https://bugzilla.redhat.com/show_bug.cgi?id=1638303

we decide that using the shortname for the cannocal server name was not valid in a triple deployment in https://bugzilla.redhat.com/show_bug.cgi?id=1957363#

after a lot of discussion in 
https://bugzilla.redhat.com/show_bug.cgi?id=1900500


so tl;dr is for adoption to work we are going to need to preserve the value of host as the FQDN

we have removed management of /etc/hosts and are relying on DNS to make hypervior_hostname be the FQDN but we need to orveride the default in all OpenStack services on the EPDM node to use the FQDN value in [DEFAULT]/hosts or we need to put the FQDN in /etc/hostsname

note that that the value of HOST and hypervior_hostname can never change for a compute node once its deployed. it on a physical server you need to scale it in and scale it back out as a new compute node.

nova explcilty has code to prevent this rename in place as it currpts the nova/neutron/cinder and cyborg databases if these ever change with running services on the node.